client/src/App.js

```diff
@@ -1,11 +1,64 @@
+import React, { useState } from "react";
 import "./App.css";
+import data from "./exampleresponse.json";
+import VideoList from "./VideoList";
+import AddVideo from "./AddVideo";
+import SearchVideo from "./SearchVideo";
 
 function App() {
+  const [VideosData, setVideosData] = useState(data);
```

Comment:
Careful with the naming here; JS variables are all `lowerCamelCase`, `UpperCamelCase` is only for `class`es.